features/ubuntu_pro.feature

```diff
@@ -90,7 +90,7 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
            | release | fips-s   | cc-eal-s | cis-s    | infra-pkg | apps-pkg |
            | xenial  | disabled | disabled | disabled | libkrad0  | jq       |
            | bionic  | disabled | n/a      | disabled | libkrad0  | bundler  |
-           | focal   | disabled | n/a      | disabled | hello     | ant      |
+           | focal   | n/a      | n/a      | disabled | hello     | ant      |
```

Comment:
(This is for AWS Pro)
Based on this spreadsheet: https://docs.google.com/spreadsheets/d/12AMMEJHaxFQBBsUbJOw6hApJG8IYPdSQEaT2yKnImu8/edit#gid=952144347
I would expect xenial to be `n/a` as well. @blackboxsw unless I'm reading it wrong or there is a better source?

Regardless, that would be a contract change, not on our end, so not blocking

Reply:
Same comment applies to azure

Reply:
That is the best source of information for FIPS support ....  xenial should disallow fips. But, there are also some beta customers that may have attempted to install a less desirable FIPS on xenial without a cloud-optimized kernel using some ua config settings like features.allow_xenial_fips_on_cloud. So we probably won't be setting the contract to disallow fips on xenial in this case